### DISCUSSION TITLE
how to hide or config tableview right click menu ?

### DISCUSSION THREAD

**Discussion by wangduanduan** [israel-dryer/ttkbootstrap#583] [Q&A]
<img width="281" alt="image" src="https://github.com/israel-dryer/ttkbootstrap/assets/16631306/a7524042-110f-4bd0-9a76-ab4d2da1d360">

**Comment by TheDemonGuard** [comment]
@wangduanduan 
If your inquiry is about disable right-click feature,
This has been reported by #730 and fixed with PR #731.

> <img alt="image" width="281" src="https://private-user-images.githubusercontent.com/16631306/335601228-a7524042-110f-4bd0-9a76-ab4d2da1d360.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE2MzMwNTAsIm5iZiI6MTc2MTYzMjc1MCwicGF0aCI6Ii8xNjYzMTMwNi8zMzU2MDEyMjgtYTc1MjQwNDItMTEwZi00YmQwLTlhNzYtYWI0ZDJkYTFkMzYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDI4VDA2MjU1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ5ZTA5MjFlMDdiZGRhMzEwNTNmOWE5NmVhYjg2ZTY3NjNhOWY3MTA0MzIzN2U2MjRiYTBlZTBmYTEyZTNmZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9H3kYb8hNlXydt1c9fExnRBhHaNYtJqg3zgc5qGi63Y">
